src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(n) {
+      this.result += n;
+    },
+
+    subtract(n) {
+      this.result -= n;
+    },
+
+    multiply(n) {
+      this.result *= n;
+    },
+
+    divide(n) {
+      this.result /= n;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, n) {
+      callback.call(this, n);
+
+      return this;
+    },
+  };
+
+  return calculator;
```

Comment:
```suggestion
  return calculator;
```
we don't need it anymore